internal/buildkitd/bundling/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Is this the buildkitd license? Makes sense if so, just checking. cc @aluzzardi, we have large chunks of code from buildkitd's main func here now, just want to double-check that including the license is all that's needed

Reply:
It is, yes